docs/sphinx/source/examples/chat_with_your_pdfs_using_colbert_langchain_and_Vespa-cloud.ipynb

```diff
@@ -40,19 +40,19 @@
                 "\n",
                 "### Connecting LangChain Retriever with Vespa for Context Retrieval from PDF Documents\n",
                 "\n",
-                "In this notebook, we seamlessly integrate a custom [LangChain](https://python.langchain.com/docs/get_started/introduction)\n",
-                "[retriever](https://python.langchain.com/docs/modules/data_connection/) with a Vespa app,\n",
+                "In this notebook, we seamlessly integrate a custom [LangChain](https://python.langchain.com/v0.1/docs/get_started/introduction)\n",
```

Comment:
Why specifically 0.1?

Reply:
That was a really good question, which prompted me to double-check if my assumption that we were still using v0.1 was still valid, which it turned out it was not. So change to v0.2 upcoming. Thanks a lot! 🙌

Reply:
Ok, the docs for v0.2 are not structured the same, so for the text and links to have coherent meaning, we need to point the links to v0.1 anyways, since thats the docs it was originally meant to refer to.